src/main/java/com/ericsson/ei/controller/RulesController.java

```diff
@@ -0,0 +1,43 @@
+
+package com.ericsson.ei.controller;
+
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * Provides interaction with resource
+ * (Generated with springmvc-raml-parser v.0.8.6)
+ * 
+ */
+@RestController
+@RequestMapping(value = "/jmespath", produces = "application/string")
+public interface RulesController {
+
+
+    /**
+     * Test rules
+     */
+	
+//   @RequestMapping(value = "", method = RequestMethod.GET)
+//	public String runRule(@RequestParam("arg1") String arg1);  
+   
```

Comment:
Remove these commented out code, if it is not going to be used in this pull request change.